packages/frontend/src/lib/post-tweet.ts

```diff
@@ -0,0 +1,7 @@
+import { clientFactory } from './twitter';
+
+export const postTweet = async (postId: string, user: string) => {
+  const url = process.env.NEXT_PUBLIC_SITE_URL + '/posts/' + postId;
```

Comment:
where is `NEXT_PUBLIC_SITE_URL` documneted? seems to not be resolving (`unknown`) in staging

Reply:
Ah sorry this is just the root url so 'http://localhost:3000' or 'https://nouns.nymz.xyz' for prod